query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/commons/LdapServiceCommons.java

```diff
@@ -273,12 +274,13 @@ private static String getLdapUrl(LdapConnectionField connection) {
   }
 
   private static URI getUriFromProperty(String ldapUrl) {
+    String url;
```

Comment:
This could be cleaned up a bit 
```
  private static URI getUriFromProperty(String ldapUrl) {
    if (StringUtils.isEmpty(ldapUrl)) {
      return null;
    }

    String newUri = PropertyResolver.resolveProperties(ldapUrl);
    if (!URI_MATCHER.matcher(newUri).matches()) {
      newUri = "ldap://" + newUri;
    }
    return URI.create(newUri);
  }
```